component/src/main/java/org/wso2/extension/siddhi/io/kafka/source/KafkaConsumerThread.java

```diff
@@ -117,21 +116,19 @@ void resume() {
 
     void restore(Map<String, Map<Integer, Long>> topicOffsetMap) {
         final Lock consumerLock = this.consumerLock;
-        if (null != topicOffsetMap) {
```

Comment:
@dnwick why have we removed the null check here?

Reply:
Ah i have missed it . Added